tests/unit/test_endpoint_consumer.py

```diff
@@ -134,11 +134,15 @@ def version(self):
         return "1.0.0"
 
 
+@unittest.mock.patch("platform.processor", lambda: "x86_64")
 class TestEndpointConsumer(unittest.TestCase):
     def setUp(self):
         metadata_file = open("metadata.yaml")
         self.harness = Harness(EndpointConsumerCharm, meta=metadata_file)
+        self.harness.add_resource("promql-transform-amd64", "dummy resource")
         self.addCleanup(self.harness.cleanup)
+        self.maxDiff = None
```

Comment:
Debugging artifact?

Reply:
yes, thanks for pointing it out. removed